server/services/store/sqlstore/migrations/000018_populate_categories.up.sql

```diff
@@ -30,6 +32,7 @@ CREATE TABLE {{.prefix}}categories (
         {{ end }}
         COALESCE(nullif(c.DisplayName, ''), 'Direct Message') as category_name,
         cm.UserId,
+        /* TODO: Ask Harshil about it */
```

Comment:
@jespino @wiggin77 any idea what the issue is here?

Reply:
We're trying to understand why we have a fake team named `direct_message` and a category named `Direct Message`.    Shouldn't these be in the default category (i.e. no category)?

Reply:
Yup, this was not needed. We shouldn't be creating categories for DM and private group messages. Fixed this to only process public and private channels and skip DM and GM.